libtiledbvcf/test/src/unit-vcf-store.cc

```diff
@@ -448,4 +448,42 @@ TEST_CASE("TileDB-VCF: Test ingest 100", "[tiledbvcf][ingest]") {
 
   if (vfs.is_dir(dataset_uri))
     vfs.remove_dir(dataset_uri);
+}
+
+TEST_CASE("TileDB-VCF: Write to existing V2 array", "[tiledbvcf][ingest][v2]") {
```

Comment:
This test modifies an existing array for purposes of ingesting new samples into an existing V2 array. It contains a shell-command to copy the array directory. I didn't want to spend the time introducing boost to do this in a safe cross-platform way because we're only running CI on Linux. If this was in the library code I would have at least used the POSIX apis, but left this as-is for a unit test.